action/flags.go

```diff
@@ -0,0 +1,166 @@
+package action
+
+// FlagHandler contains all flags from cmd
+type FlagHandler struct {
+	Bench                bool   `json:"bench"`
+	ExpectSize           int64  `json:"expect_size"`
+	Format               string `json:"format"`
+	HumanReadable        bool   `json:"human_readable"`
+	LongFormat           bool   `json:"long_format"`
+	MaximumMemoryContent int64  `json:"maximum_memory_content"`
+	Recursive            bool   `json:"recursive"`
```

Comment:
The same here.

Actually, I don't think we need a `FlagHandler` at all, only `bench` and `zone` will be shared, others should all be moved. And even we have a `FlagHandler` here, we still need to implement `WithZone` for every `XXXHandler`.

Reply:
> The same here.
> 
> Actually, I don't think we need a `FlagHandler` at all, only `bench` and `zone` will be shared, others should all be moved. And even we have a `FlagHandler` here, we still need to implement `WithZone` for every `XXXHandler`.

At the very beginning, I try to abstract all the flags for better reusing. However, I find that I still have to rewrite the `WithXXX` method for every `XXXHandler` like you mentioned before. So, I agree with you, and will remove `FlagHandler` in next commit.